pkg/driver/kubernetes/factory.go

```diff
@@ -178,6 +181,15 @@ func (d *Driver) initDriverFromConfig() error {
 			deploymentOpt.ContainerRuntime = v
 		case "custom-config":
 			deploymentOpt.CustomConfig = v
+		case "env":
+			// Split over comma for multiple key/value
+			re := regexp.MustCompile(`([^=]+)=([^=]+)`)
```

Comment:
Since you're using ';' to delimit the combined string, I'm curious why you're rejecting a "=" in the value.  Maybe I'm being pedantic, but given there are a lot of layers in BuildKit and the libraries it consumes, I'm not sure if there's some scenario where someone might need to pass in a value that contains an equal sign.  Could we instead just perform a `strings.SplitN(item, "=", 2)` around line 188?

Reply:
OK, I will do that. Its sound a good idea